client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -410,6 +412,10 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 }
 
 const mapStateToProps = ( state, { postId, postType, duplicatePostId }: Props ) => {
+	const statsPath = postId
```

Comment:
I expect that `mapStateToProps` only return props that depend on the state, but it's not the case of both `statsPath` and `statsProps`. Why don't we move this logic directly to the render method?

Reply:
Oh! That's a great distinction; I've always just thought of `mapStateToProps` as where we prepare props to pass into the component (computed on state or otherwise). But of course – it's right there in the name 😜. Thanks for pointing that out 👍